Mathlib/CategoryTheory/Limits/VanKampen.lean

```diff
@@ -570,9 +570,9 @@ theorem BinaryCofan.isVanKampen_mk {X Y : C} (c : BinaryCofan X Y)
     have he₁ : c'.inl = e₁.hom ≫ (cones f c.inl).fst := by simp [e₁]
     have he₂ : c'.inr = e₂.hom ≫ (cones f c.inr).fst := by simp [e₂]
     rw [he₁, he₂]
-    apply BinaryCofan.isColimitCompRightIso (BinaryCofan.mk _ _)
-    apply BinaryCofan.isColimitCompLeftIso (BinaryCofan.mk _ _)
-    exact h₂ f
+    exact
+      (BinaryCofan.mk _ _).isColimitCompRightIso e₂.hom
+        ((BinaryCofan.mk _ _).isColimitCompLeftIso e₁.hom (h₂ f))
```

Comment:
Optional: I tend to write this as
```suggestion
    exact (BinaryCofan.mk _ _).isColimitCompRightIso e₂.hom
      ((BinaryCofan.mk _ _).isColimitCompLeftIso e₁.hom (h₂ f))

Reply:
I mildly lean this way also but leave it up to you.